src/main/java/org/broadinstitute/hellbender/engine/filters/MateDistantReadFilter.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.ReadFilterArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Keep only paired reads that are not near each other in a coordinate-sorted source of reads.
+ * (I.e., they're both mapped, but mapped to different contigs, or are mapped too far apart on the same contig.)
+ *
+ * <p>See MappedReadFilter for criteria defining an mapped read.</p>
+ */
+@DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY, summary = "Keep only paired reads with mates mapped >=1KB apart or on different contigs", extraDocs = ReadFilterLibrary.MappedReadFilter.class)
```

Comment:
In the summary change ">=1KB" to ">= mate-too-distant-length (default 1KB)"

Reply:
done